content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -90,6 +90,7 @@ For a reference to old feature gates that are removed, please refer to
 | `CustomResourceValidationExpressions` | `true` | Beta | 1.25 | |
 | `DelegateFSGroupToCSIDriver` | `false` | Alpha | 1.22 | 1.22 |
 | `DelegateFSGroupToCSIDriver` | `true` | Beta | 1.23 | |
+| `DelegateFSGroupToCSIDriver` | `true` | GA | 1.26 |-|
```

Comment:
A GA'ed feature should be moved to the second table on this page.

Reply:
Done